src/Auth/File/UserGroup.php

```diff
@@ -9,6 +9,6 @@ class UserGroup extends BaseUserGroup
 {
     public function queryUsers()
     {
-        return User::query()->where('groups/'.$this->handle(), true);
+        return User::query()->where('groups->'.$this->handle(), true);
```

Comment:
I would rather not do this as it would be a breaking change.

Anyone doing `->where('groups/foo')` would need to change to `->where('groups->foo')`.

I think a better solution would be to escape the slashes when we go to add the items in the cache.

In this line, we could (also) replace slashes with something else: https://github.com/statamic/cms/blob/f6b573ee5046b55f75bb450c69fe2ec5529eca34/src/Stache/Indexes/Index.php#L131

Reply:
Ok I've opted for `->` to keep it consistent with whats been done with JSON fields